pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pdftext"
-version = "0.3.20"
+version = "0.3.21"
```

Comment:
This should be a minor version bump, to 0.4

Reply:
Done!